code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -72,6 +72,13 @@
 	if(istype(mymob) && mymob.client?.prefs?.read_preference(/datum/preference/toggle/ambient_occlusion))
 		add_filter("AO", 1, drop_shadow_filter(x = 0, y = -2, size = 4, color = "#04080FAA"))
 
+/atom/movable/screen/plane_master/game_world/fov_hidden
```

Comment:
Stuff added in this file can probably go in modular_skyrat/master_files/code/_onclick/hud/rendering/plane_master.dm